exotica_core/include/exotica_core/property.h

```diff
@@ -129,11 +129,6 @@ class Instantiable : public virtual InstantiableBase
         parameters_ = init;
     }
 
-    [[deprecated]] virtual void Instantiate(C& init)
-    {
-        Instantiate(static_cast<const C&>(init));
-    }
```

Comment:
Does this mean that we will not be backward compatible and all old problems/tasks need to adapt to the new virtual method?

Reply:
Yes, and I carried it through all solvers I have access to and currently in the workspace (OPT++, SNOPT, KNITRO, IPOPT, etc.). If there's anything I have missed, I am happy to fix it if I get a pointer.